Sources/RemoteMessaging/RemoteMessageRequest.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  RemoteMessageRequest.swift
+//
+//  Copyright © 2022 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+import Networking
+
+public struct RemoteMessageRequest {
+
+    public let endpoint: URL
+
+    public init(endpoint: URL) {
```

Comment:
This struct used to hardcode the endpoint. Now, in order to prepare for reuse by iOS and macOS, it's parametrized with the endpoint URL, provided by the client app.